components/__test__/AnchoredHeading.test.tsx

```diff
@@ -0,0 +1,47 @@
+import AnchoredHeading from '../AnchoredHeading';
```

Comment:
Let's not add tests in this PR, just the infrastructure behind testing.

Reply:
If not put no test it returns an error of type `1`.

Reply:
Nope,you can pass `--passWithNoTests` on package.json